perl/README.md

```diff
@@ -0,0 +1,1021 @@
+# Bystro High Dimensional Genetics Annotator Documentation
```

Comment:
What exactly do you mean by high dimensional? WGS or a lot of annotations or both? Does the average geneticist understand high dimensional in general?

Reply:
I think 'genomics' instead of 'genetics' would also make more sense if you're trying to express that it is 'high dimensional' if you mean that the genetics are the high dimension.
Bystro Genomics Annotator?

Reply:
I can rename to genomics, but high dimensional indicates big data. Here it is very appropriate, because dimensionality refers to the 'p' in n * p, and we can output a very large p due to the performance of the annotator.